consensus/consortium/v2/consortium_test.go

```diff
@@ -2532,3 +2536,77 @@ func TestEncodeDecodeValidatorBitSet(t *testing.T) {
 		}
 	}
 }
+
+func randFieldElement() [32]byte {
+	bytes := make([]byte, 32)
+	_, err := rand.Read(bytes)
+	if err != nil {
+		panic("failed to get random field element")
+	}
+	var r fr.Element
+	r.SetBytes(bytes)
+
+	return gokzg4844.SerializeScalar(r)
+}
+
+func randBlob() *kzg4844.Blob {
+	var blob kzg4844.Blob
+	for i := 0; i < len(blob); i += gokzg4844.SerializedScalarSize {
+		fieldElementBytes := randFieldElement()
+		copy(blob[i:i+gokzg4844.SerializedScalarSize], fieldElementBytes[:])
+	}
+	return &blob
+}
+
+func TestVerifyBlobHeader(t *testing.T) {
```

Comment:
I think this test does not have a good coverage. skipBlobCheck will return true and we don't reach the below checks. In unit test for this case, you should create an invalid version hash, check if it fails as expected. Then correct the version hash, check both 2 cases that skipBlobCheck returns true and false. And continue doing the same for the following checks.

Reply:
I have added more tests to cover all cases